src/App.js

```diff
@@ -1,5 +1,41 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "../src/constants/messages.js";
+import carInputValidation from "./utils/carInputValidation.js";
+import countInputValidation from "./utils/countInputValidation.js";
+import progress from "./utils/progress.js";
+import racePrint from "./utils/racePrint.js";
+import winner from "./utils/winner.js";
 class App {
-  async play() {}
+  async play() {
+    MissionUtils.Console.print(MESSAGES.START);
+    const CARS_NAMES = await MissionUtils.Console.readLineAsync("");
```

Comment:
```suggestion
    const CARS_NAMES = await MissionUtils.Console.readLineAsync(MESSAGES.START);
```
여기서 `print`하는 로직과 유저 인풋을 받는 로직을 나누지 않고 `readLineAsync` 함수의 인자로 넣어주는건 어떠신가요??

Reply:
아 맞네요 !! 수정하겠습니당~